native/cocos/platform/android/AndroidPlatform.cpp

```diff
@@ -204,6 +204,8 @@ class GameInputProxy {
 
     bool cookGameActivityMotionEvent(GameActivityMotionEvent *motionEvent) {
         if (motionEvent->pointerCount > 0) {
+            touchEvent.windowId = 1; // Assuming main window
```

Comment:
Should send to corresponding window. Why can assume main window?

Reply:
In the function `cookGameActivityMotionEvent`,  all events are sent from the main window, so the windowId is 1.